debezium-quarkus/extensions/outbox/runtime/src/main/java/io/debezium/outbox/quarkus/ExportedEvent.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.outbox.quarkus;
+
+import java.time.Instant;
+
+/**
+ * Describes an event that should be exported via the "outbox" table.
+ *
+ * @author Chris Cranford
+ */
+public interface ExportedEvent<I, P> {
```

Comment:
Could you add the `@Incubating` annotation?

Reply:
Added